src/translations.php

```diff
@@ -34,11 +34,21 @@
         "Present" => "Present",
     ],
     // Locales below are sorted alphabetically
+    "ar" => [
+        "Total Contributions" => "إجمالي المساهمات",
```

Comment:
`"rtl" => true` can be used to mark it as a right to left language which should help with rtl/ltr run ordering

```suggestion
        "rtl" => true,
        "Total Contributions" => "إجمالي المساهمات",
```

Reply:
Ah, my mistake, as I hadn't noticed that previously! I'll make sure to implement that in for future translations! Thanks for the check-in!

Reply:
Feel free to use the "commit suggestion" button to co-author the fix.